.github/workflows/e2e.yml

```diff
@@ -49,6 +49,13 @@ jobs:
           layerform config set-context test-cloud -t cloud --url https://demo.layerform.dev --email foo@bar.com --password strongpass
           layerform config set-context test-local -t local --dir test
 
+      - name: layerform config select-context
+        run: |
+          # fails if command succeeds
+          ! layerform config select-context test-does-not-exist # context does not exist
+
+          layerform config set-context test-local -t local --dir test && layerform config select-context test-local
```

Comment:
I'm still wondering what is the default layerform/configurations.yml file in our CI/CD testing env.

As the mitigation to ensure that a config exists, I do `set-context` first to create a config.

Reply:
Well, it is the last one that run in the previous step (check line 50).
At this point, the current context is the `test-local`, your `set-context` ends up being a `no-op`.
What I think you should do to test is somehow check if the `select-context` actually works.
You can use the `config get-contexts` command alongside with `grep` for that. The `selected` context will have a `*` in it's line.
I'll make a comment with suggestions.

Reply:
updated https://github.com/ergomake/layerform/pull/96/commits/9e15b350bd653ff464027be46301bdaeb02d878e